features/corporate_page.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Settings
+  As a corporate partner
+  I want to be able to view more information on partnering
+  so that I am aware of different ways of partnering with MAS.
+
+  @enable-corporate
+  Scenario: Accessing the corporate page
+    When  I visit the corporate page
+    Then  I should be informed that I am on the correct page
+
+  Scenario: Corporate page is not accessible when feature is disabled
+    When I visit the corporate page
```

Comment:
If we are going to align the non-gherkin reserved stuff (which I don't necessarily agree with myself, but it seems other folks like to), then it should be consistent. This one is out a little.